### PR TITLE
Add pulse detection for MMA8451

### PR DESCRIPTION
**Pull request by jablan**
Sorry for the whitespace change, my editor is set up to delete the trailing whitespace upon save. Tell me if you would prefer me to roll back the whitespace changes.

If you are OK with the change, I'd also add the documentation if needed.

### CODE REVIEW

File: AccelerometerMMA8451/AccelerometerMMA8451.h [hunk 1]
```diff
@@ -1234,6 +1241,16 @@ class AccelerometerMMA8451: public Accelerometer, public RegisterBasedWiredDevic
      */
     void setTransientCount(unsigned char count);
 
+    void setPulseDetection(bool enable, unsigned char axis, bool dpa);
```
Comment (dalmirdasilva) [inline AccelerometerMMA8451/AccelerometerMMA8451.h:1255]: nit: can you provide some comments on these new methods?

Comment (dalmirdasilva) [reply AccelerometerMMA8451/AccelerometerMMA8451.h:1255]: White spaces is fine. You can add your name into the AUTHORS files if you want too.

Comment (jablan) [reply AccelerometerMMA8451/AccelerometerMMA8451.h:1255]: Ok, I have provided some comments on the new methods, mostly by copy-pasting corresponding descriptions from AN4072. I must admit I don't understand most of them, except for the most basic stuff, but the methods are there.

**Review by dalmirdasilva — APPROVED** [review]
LGTM

Thanks!

**Pull request merged**